grammars/jsdoc.cson

```diff
@@ -411,6 +435,120 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Double-quoted string
+      {
+        'begin': '(")'
```

Comment:
To stay consistent with JavaScript. Furthermore, some themes might colour double-quoted strings differently to single-quoted strings ([such as `seti-syntax`](https://github.com/jesseweed/seti-syntax/blob/29fc4f4/styles/syntax.less#L347-L350)).